docs/pages/management/export-audit-events/datadog.mdx

```diff
@@ -0,0 +1,412 @@
+```
+
+## Step 4/6. Create teleport-event-handler credentials
+
+### Enable impersonation of the Event Handler user
+
+In order for the Teleport Event Handler plugin to forward events from your Teleport
+cluster, it needs a signed identity file from the cluster's certificate authority.
+The `teleport-event-handler` user cannot request this itself, and requires another
+user to **impersonate** this account in order to request a certificate.
+
+Create a role that enables your user to impersonate the Fluentd user. First,
+paste the following YAML document into a file called
+`teleport-event-handler-impersonator.yaml`:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: teleport-event-handler-impersonator
+spec:
+  # SSH options used for user sessions
+  options:
+    # max_session_ttl defines the TTL (time to live) of SSH certificates
+    # issued to the users with this role.
+    max_session_ttl: 10h
+
+  # allow section declares a list of resource/verb combinations that are
+  # allowed for the users of this role. by default nothing is allowed.
+  allow:
+    impersonate:
+      users: ["teleport-event-handler"]
+      roles: ["teleport-event-handler"]
+```
+
+Next, create the role:
+
+```code
+$ tctl create -f teleport-event-handler-impersonator.yaml
+```
+
+(!docs/pages/includes/add-role-to-user.mdx role="teleport-event-handler-impersonator"!)
+
+### Export an identity file for the Event Handler plugin user
+
+The Teleport Event Handler plugin uses the `teleport-event-handler` role and user to
+read events. We export an identity file for the user with the `tctl auth sign`
+command.
+
+<Tabs>
+<TabItem label="Executable" scope={["oss","enterprise"]}>
+```code
+$ tctl auth sign --user=teleport-event-handler --out=identity
+```
+
+This command creates one PEM-encoded file, `identity`. The identity file
+includes both TLS and SSH credentials. The Event Handler plugin uses the SSH
+credentials to connect to the Proxy Service, which establishes a reverse tunnel
+connection to the Auth Service. The plugin uses this reverse tunnel, along with
+your TLS credentials, to connect to the Auth Service's gRPC endpoint.
+
+</TabItem>
+<TabItem label="Helm Chart" scope={["cloud"]}>
+
+If you are planning to use the Helm Chart, you'll need to generate the keys
+with the `file` format, then create a secret in Kubernetes.
+
+Create the identity using the following command:
+
+```code
+$ tctl auth sign --format=file --user=teleport-event-handler --out=identity
+```
+
+Then create the Kubernetes secret:
+
+```code
+$ kubectl create secret generic teleport-event-handler-identity --from-file=auth_id=identity
+```
+
+These commands should result in a PEM-encoded file, `identity`, and a secret
+in Kubernetes with the name `teleport-event-handler-identity`.
+
+</TabItem>
+</Tabs>
+
+## Step 5/6. Install Fluentd output plugin for Datadog
+
+In order for Fluentd to communicate with Datadog, it requires the [Fluentd output
```

Comment:
I'd like to push back on this a bit. I know what our style guide says, but putting myself in the position of a reader, I'd like an opportunity to review a piece of third party software instead of blindly installing in into my system.

Also consider the future where the instructions below might go stale, and until we know/resolve it the first party docs should (finger crossed) update with changes.

Reply:
In that case, would it make sense to tell the reader to follow the installation instructions in the `fluent-plugin-datadog` README?

Reply:
Not for the other reader who _doesn't_ want to review their documentation, or already has and is following this guide to recreate on another environment, etc. This is a middle-ground where the user can _choose_ to follow the link but doesn't have to, and our docs contain a reasonable amount of content for a third party system.